talks/storage-requirements/storage-requirements.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Storage requirements"
+author: "Florian Ziemen"
+subtitle: ""
+format:
+  revealjs:
+    transition: slide
+    slide-number: true
+    code-background: true
+    logo: /static/logos.png
+    footer: "[Talks page](/talks)"
+    theme: [default, ../custom.scss]
+    chalkboard:
+      buttons: true
+    navigation-mode: linear
+    auto-stretch: true
+    center: true
+    mermaid:
+      theme: default
+---
+
+
+# What goes where
+
+* Healpix data for variables requested is duplicated among those centers that have enough space.
+* Native grid data stays at the originating center.
+* Bonus variables stay at the originating center.
+
+# Requirements for Healpix data
+
+## Ballpark estimate for level 9 (13km)
+
+| type   | cells / snapshot | snapshots | GB / var [^1] | vars requested | GB total |
+| ------ | ---------------- | --------- | ------------- | -------------- | -------- |
+| 2D     | 3 M              | 365*24    | 52            | 33             | 1650     |
+| 2D[^2] | 3 M              | 365*4     | 13            | 33             | 275      |
+| 3D     | 75 M             | 365*4     | 220           | 12             | 2600     |
```

Comment:
This is almost the table I was referring to above--would you mind adding a `GB / snapshot` column?